doc/source/examples/rpt/tuning-parameters-with-nomad/tuning-parameters-with-nomad.rst

```diff
@@ -0,0 +1,320 @@
+
+Before executing the ``rpt_3d`` application we have to change a few parameters in the ``rpt_parameters.prm`` file.
+
+First, in the *"rpt parameters"* subsection, we may set the ``verbosity`` parameter to ``verbose``, set the ``export counts`` parameter to ``true``, and change the name of the ``counts file`` that will be exported if we wish.
+
+Second, in the *"parameter tuning"* subsection, we have to set the ``tuning`` parameter to ``false`` since we're trying to tune parameters anymore.
+
+Lastly, in the *"detector parameters"* subsection, we have to change the values of the parameters that we tuned (``dead time``, ``activity``, and ``attenuation coefficient reactor``) with the ones NOMAD gave us.
+
+By doing the modifications mentioned above, the content ``rpt_parameters.prm`` should look like this :
+
+.. code-block:: text
+
+    # Listing of Parameters
+    # ---------------------
+    # --------------------------------------------------
+    # RPT Monte Carlo technique
+    #---------------------------------------------------
+    subsection rpt parameters
+        set particle positions file          = positions.particle
+        set verbosity                        = verbose
+        set export counts                    = true
+        set counts file                      = counts_calculated.csv
+        set monte carlo iteration            = 100000
+        set random number seed               = 0
+        set reactor height                   = 0.3
+        set reactor radius                   = 0.4
+        set peak-to-total ratio              = 0.4
+        set sampling time                    = 0.01
+        set gamma-rays emitted               = 2
+        set attenuation coefficient detector = 21.477
+    end
+
+    # --------------------------------------------------
+    # Tuning with NOMAD
+    #---------------------------------------------------
+    subsection parameter tuning
+        set tuning                           = false
+        set cost function type               = larachi
+        set experimental data file           = counts.experimental
+    end
+
+    #---------------------------------------------------
+    # Detector parameters
+    #---------------------------------------------------
+    subsection detector parameters
+        set detector positions file          = positions.detector
+        set radius                           = 0.0381
+        set length                           = 0.0762
+        set dead time                        = 7.85479e-06
+        set activity                         = 2.43045e+06
+        set attenuation coefficient reactor  = 0.5002
+    end
+
+In the figure below, we can see that there is very little difference between the experimental counts and the calculated counts with the tuned parameters.
```

Comment:
Amishga :) 
This would be great if you add the linear fit graph for calculated count and experimental count and report the R-Squared value as well to be more quantitive in reporting the result.

Reply:
Awesome idea.